android/src/main/java/com/twiliovoicereactnative/ReactNativeArgumentsSerializer.java

```diff
@@ -206,21 +209,18 @@ public static WritableArray serializeAudioDeviceMapIntoArray(Map<String, AudioDe
   public static WritableMap serializeAudioDeviceInfo(
     Map<String, AudioDevice> audioDevices,
     String selectedAudioDeviceUuid,
-    AudioDevice selectedAudioDevice
-  ) {
-    WritableMap audioDevicesInfo = Arguments.createMap();
-
-    audioDevicesInfo.putArray(AudioDeviceKeyAudioDevices, serializeAudioDeviceMapIntoArray(audioDevices));
-    if (selectedAudioDevice != null) {
-      audioDevicesInfo.putMap(AudioDeviceKeySelectedDevice, serializeAudioDevice(selectedAudioDeviceUuid, selectedAudioDevice));
-    }
-
-    return audioDevicesInfo;
-  }
-  public static WritableMap serializeVoiceException(@NonNull VoiceException exception) {
+    AudioDevice selectedAudioDevice) {
     return constructJSMap(
-      new Pair<>(VoiceErrorKeyCode, exception.getErrorCode()),
-      new Pair<>(VoiceErrorKeyMessage, exception.getMessage()));
+      new Pair<>(AudioDeviceKeyAudioDevices, serializeAudioDeviceMapIntoArray(audioDevices)),
+      new Pair<>(AudioDeviceKeySelectedDevice, serializeAudioDevice(selectedAudioDeviceUuid, selectedAudioDevice)));
```

Comment:
Same with this line, could be explicit `null` now.

Reply:
look above